tests/test-sources/modules/performance/byte-compile-lua.nix

```
@@ -1,5 +1,7 @@
-{ pkgs, ... }:
+{ lib, pkgs, ... }:
 let
+  pluginStubs = import ../../../utils/plugin-stubs.nix { inherit lib pkgs; };
```

As per above suggestion

```suggestion
  pluginStubs = pkgs.callPackages ../../../utils/plugin-stubs.nix { };
```